Server/JokesOnYou.Web.Api/Controllers/JokesController.cs

```diff
@@ -21,6 +22,16 @@ public JokesController(IJokesService jokesService)
             _jokesService = jokesService;
         }
 
+        [Authorize(Roles = "Registered,Admin")]
```

Comment:
idea: maybe member instead of registered?

Reply:
Yea thats fine, this is just placeholder anyway ;)